persistence/src/main/java/com/zebrunner/reporting/persistence/dao/mysql/application/TestMapper.java

```diff
@@ -27,7 +28,9 @@
 
     List<Test> getTestsByTestRunId(long testRunId);
 
-    List<TestResult> getTestResultsByTestCaseId(@Param("testCaseId") Long testCaseId, @Param("limit") Long limit);
+    List<TestResult> getTestResultsByStartTimeAndTestCaseId(@Param("testCaseId") Long testCaseId,
+                                                            @Param("startTime") Date startTime,
```

Comment:
Why Data instead of LocalDateTime?

Reply:
Because value is extracted from Test object, where we use date